src/testConfigManager.ts

```diff
@@ -16,21 +16,30 @@ class TestConfigManager {
         return this.configRelativePath;
     }
 
-    public async loadRunConfig(tests: ITestItem[], isDebug: boolean): Promise<IExecutionConfigGroup[]> {
-        const folderSet: Set<WorkspaceFolder> = this.getFoldersOfTests(tests);
-        const configs: ITestConfig[] = [];
-        for (const folder of folderSet.values()) {
-            const configFullPath: string = path.join(folder.uri.fsPath, this.configRelativePath);
-            if (!await fse.pathExists(configFullPath)) {
-                continue;
+    public async loadRunConfig(tests: ITestItem[], isDebug: boolean, isDefaultConfig: boolean): Promise<IExecutionConfig | undefined> {
+        // TODO: Handle multi-root
+        const configs: IExecutionConfig[] | undefined = workspace.getConfiguration('java.test').get<IExecutionConfig[]>('config');
+        if (configs && configs.length > 0) {
+            // Use the new config schema
+            if (isDefaultConfig) {
```

Comment:
`isDefaultConfig` is kind of confusing. something like `usingDefaultConfig` would be better to understand.